checker/check_result.go

```diff
@@ -118,6 +118,35 @@ type CheckResult struct {
 	Details2 []CheckDetail `json:"-"` // Details of tests and sub-checks
 	Score    int           `json:"-"` // {[-1,0...10], -1 = Inconclusive}
 	Reason   string        `json:"-"` // A sentence describing the check result (score, etc)
+
+	// UPGRADEv6
+	// Stores raw data results.
+	// We re-use Error2 and Version.
+	RawResults *RawResults `json:"-"`
```

Comment:
Why do we need to copy over the `RawResults` ptr to each `CheckResult`? The ptr passed in `CheckRequest` is what gets updated right?

Reply:
I think you're right, may not be needed. It's there in the code because of the original PR using `Raw interface{}`. I'll update, thanks!

Reply:
done.